apps/framework-docs/src/pages/stack/workflows/schedule-workflow.mdx

```diff
@@ -0,0 +1,306 @@
+| `@every 5m` | Every 5 minutes |
+| `@every 1h` | Every hour |
+| `@every 12h` | Every 12 hours |
+| `@every 24h` | Every 24 hours |
+| `@every 7d` | Every 7 days |
+
+## Practical Scheduling Examples
+
+### Daily Data Processing
+
+<TypeScript>
+```typescript filename="app/daily-etl.ts" copy
+import { Task, Workflow } from "@514labs/moose-lib";
+
+export const dailyDataProcessing = new Workflow("daily-data-processing", {
+  startingTask: extractDataTask,
+  schedule: "0 2 * * *", // Run at 2 AM every day
+  retries: 2,
+  timeout: "2h",
+});
+```
+</TypeScript>
+
+<Python>
+```python filename="app/daily_etl.py" copy
+from moose_lib import Workflow, WorkflowConfig
+
+daily_data_processing = Workflow(
+  name="daily-data-processing",
+  config=WorkflowConfig(
+    starting_task=extract_data_task,
+    schedule="0 2 * * *",  # Run at 2 AM every day
+    retries=2,
+    timeout="2h"
+  )
+)
+```
+</Python>
+
+### Weekly Reports
+
+<TypeScript>
+```typescript filename="app/weekly-reports.ts" copy
+export const weeklyReports = new Workflow("weekly-reports", {
+  startingTask: generateReportTask,
+  schedule: "0 9 * * 1", // Run at 9 AM every Monday
+  retries: 1,
+  timeout: "1h",
+});
+```
+</TypeScript>
+
+<Python>
+```python filename="app/weekly_reports.py" copy
+weekly_reports = Workflow(
+  name="weekly-reports",
+  config=WorkflowConfig(
+    starting_task=generate_report_task,
+    schedule="0 9 * * 1",  # Run at 9 AM every Monday
+    retries=1,
+    timeout="1h"
+  )
+)
+```
+</Python>
+
+### High-Frequency Monitoring
+
+<TypeScript>
+```typescript filename="app/monitoring.ts" copy
+export const systemMonitoring = new Workflow("system-monitoring", {
+  startingTask: checkSystemHealthTask,
+  schedule: "@every 5m", // Check every 5 minutes
+  retries: 0, // Don't retry monitoring checks
+  timeout: "30s",
+});
+```
+</TypeScript>
+
+<Python>
+```python filename="app/monitoring.py" copy
+system_monitoring = Workflow(
+  name="system-monitoring",
+  config=WorkflowConfig(
+    starting_task=check_system_health_task,
+    schedule="@every 5m",  # Check every 5 minutes
+    retries=0,  # Don't retry monitoring checks
+    timeout="30s"
+  )
+)
+```
+</Python>
+
+## Monitoring Scheduled Workflows
+
+### Development Environment
+
+If your dev server is running, you should see logs in the terminal when your scheduled workflow is executed:
+
+```bash filename="Terminal" copy
+moose dev
+```
+
+```txt filename="Terminal"
+[2024-01-15 12:00:00] Scheduled workflow 'daily-data-processing' started
+[2024-01-15 12:00:01] Task 'extract' completed successfully
+[2024-01-15 12:00:15] Task 'transform' completed successfully
+[2024-01-15 12:00:30] Task 'load' completed successfully
+[2024-01-15 12:00:30] Workflow 'daily-data-processing' completed successfully
+```
+
+### Checking Workflow Status
+
+You can check the status of scheduled workflows using the CLI:
+
+```bash filename="Terminal" copy
+# List all workflow executions
+moose workflow list
```

Comment:
Workflow executions have been moved to `moose workflow history` so it's more clear that it's your list of executions.

`moose workflow list` shows the workflows that we know about. It has the same output as `moose ls --type workflows`